tests/features/drupal-login.feature

```diff
@@ -0,0 +1,11 @@
+@api
+Feature: Login through Drupal
+  If configured properly
+  I can access the CMS backend through Drupal
+
+  Scenario: If configured properly I can access the CMS backend through Drupal
+    Given the site is configured to use Drupal login
+    And I am logged in as a user with the "authenticated" role
+    Then I should see the link "Log out"
+    When I click "Log out"
+    Then I should not see the link "Log out"
```

Comment:
How sure are we that we are back on the site and everything is fine. Not seeing "Log out" does not mean that we actually logged out right?

Reply:
Falls outside the scope of this ticket.